tests/testthat/test-copy-to.R

```diff
@@ -12,31 +12,6 @@ test_that("sdf_copy_to works for default serializer", {
   )
 })
 
-test_that("sdf_copy_to works for scala serializer", {
-  skip_livy()
-
-  df <- matrix(0, ncol = 5, nrow = 2) %>% dplyr::as_tibble(.name_repair = "unique")
-  df_tbl <- sdf_copy_to(sc, df, overwrite = TRUE, serializer = "csv_file_scala")
```

Comment:
any test case specifying `serializer = "csv_*"` will be no longer applicable after this change